student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name, parent_permission)
```

Comment:
- [ ] [OPTIONAL] I would suggest you set parent_permission to true. As an example:
>   def initialize(age, classroom, name = "Unknown", parent_permission = true)